### PR TITLE
Orgs: Add future deprecation notice

### PR DESCRIPTION
**Pull request by torkelo**
* [ ] Refine wording
* [ ] Create an issue to link to for feedback. The issue should have a little bit more detail but messaging should basically say the same thing as the infobox. 
* [ ] Add deprecation info to changelog

Screenshot from 2020-05-11 11-37-02

### CODE REVIEW

File: public/app/features/admin/AdminListOrgsPage.tsx [hunk 1]
```diff
@@ -29,6 +29,22 @@ export const AdminListOrgsPages: FC = () => {
       <Page.Contents>
         <>
           <div className="page-action-bar">
+            <InfoBox branded>
+              <VerticalGroup spacing="xs">
+                <Badge text="Info" color="blue" icon="info-circle" />
```
Comment (bergquist) [inline public/app/features/admin/AdminListOrgsPage.tsx:34]:
```suggestion
                <Badge text="Warning" color="blue" icon="info-circle" />
```
Im not sure if warnings exists but I think the "level" should be higher.

File: public/app/features/admin/AdminListOrgsPage.tsx [hunk 2]
```diff
@@ -29,6 +29,22 @@ export const AdminListOrgsPages: FC = () => {
...
+                <p>
+                  Organizations is a feature that only 1% of Grafana users are utilizing, and we feel that most of those
+                  users would have a better experience using Teams instead. As such, we are considering de-emphasizing
+                  and eventually deprecating Organizations in future Grafana releases. If you would like to provide
+                  feedback or describe your use case, please do so{' '}
+                  <a className="external-link" href="https://github.com/grafana/grafana/issues/xxx">
```
Comment (bergquist) [inline public/app/features/admin/AdminListOrgsPage.tsx:40]: Could you please create this issue now? Would be nice to have a real issue to link to so we don't forget to update it :)

Comment (bergquist) [reply public/app/features/admin/AdminListOrgsPage.tsx:40]: The issue can be named placeholder for now since it doesn't affect the link.

**Review by bergquist — COMMENTED** [review]
:+1:

**Comment by dprokop** [comment]
@torkelo not a fan of using the branded info box for this purpose. I was thinking about it as a way of bringing playful attention to new features we want to highlight.

For deprecation warnings I think we should use something more "serious" and toned down visually. Maybe @jessover9000 can help? Or just use regular info box

**Comment by torkelo** [comment]
But this is supposed to be a bit friendly so I think this box looks fine as, do not like the old infobox, this is so much better

File: pkg/services/sqlstore/sqlstore.go [hunk 3]
```diff
@@ -101,13 +101,42 @@ func (ss *SqlStore) Init() error {
...
+
+	return ss.WithDbSession(context.Background(), func(session *DBSession) error {
+		resp := make([]*targetCount, 0)
+		if err := session.SQL("select count(id) as Count from org").Find(&resp); err != nil {
+			return err
+		}
+
+		if resp[0].Count > 1 {
+			ss.log.Warn(`[Deprecation notice]`)
+			ss.log.Warn(`Organizations is a feature that only 1% of Grafana users are utilizing, and we feel that most of those`)
```
Comment (ryantxu) [inline pkg/services/sqlstore/sqlstore.go:129]:
```suggestion
			ss.log.Warn(`Organizations is a feature that only 1% of Grafana users are using, and we feel that most of those`)
```

**Comment by ryantxu** [comment]
The sentance is full of "use, user, utilize, using" can we try some alternatives?

File: public/app/features/admin/AdminListOrgsPage.tsx (hunk 1 above)
Comment (torkelo) [reply public/app/features/admin/AdminListOrgsPage.tsx]: removed the badge, the warnings and red color look too strong

File: public/app/features/admin/AdminListOrgsPage.tsx (hunk 2 above)
Comment (torkelo) [reply public/app/features/admin/AdminListOrgsPage.tsx]: added issue link

**Comment by torkelo** [comment]
Updated text and added issue link, https://github.com/grafana/grafana/issues/24588
Screenshot from 2020-05-12 18-37-25

**Review by bergquist — APPROVED** [review]

**Pull request merged**